ansible/roles/contiv/tasks/netmaster.yml

```diff
@@ -0,0 +1,29 @@
+
+- name: copy netmaster exec file
+  copy: src={{ contiv_bin_path }}/netmaster dest=/usr/bin/netmaster mode=0755
+
+- name: copy netctl exec file
+  copy: src={{ contiv_bin_path }}/netctl dest=/usr/bin/netctl mode=0755
+
+- name: copy netmaster setup script
+  template: src=nw_setup.j2 dest=/usr/bin/nw_setup.sh mode=0755
+
+- name: copy environment file for netmaster
+  copy: src=netmaster.env dest=/etc/default/netmaster
+
+- name: copy systemd units for netmaster
+  copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
+
+- name: setup netmaster host alias
+  lineinfile: dest=/etc/hosts line="{{ contiv_service_vip }} netmaster"
+
+- name: restart netmaster
+  service: name=netmaster state=restarted
+
+- name: wait for netmaster
```

Comment:
Is this needed? I believe service: name=netmaster state=restarted will fail if the restart is not successful. Unusable runs are long, so we want to everything we can to make things quick.